apps/client/src/features/rundown/Rundown.tsx

```diff
@@ -133,7 +143,16 @@ export default function Rundown(props: RundownProps) {
         }
       }
     },
-    [cursor, entries, insertAtCursor, moveCursorTo],
+    [cursor, entries, insertAtCursor, setEditMode],
+  );
+
+  const handleKeyUp = useCallback(
+    (event: KeyboardEvent) => {
+      if (['ShiftLeft', 'ShiftRight', 'ControlLeft', 'ControlRight', 'OSLeft', 'OSRight'].includes(event.code)) {
```

Comment:
Ugly code, but it's the simplest without some abstraction

Reply:
If you use the key instead of the code, Shift can replace ShiftLeft and ShiftRight, same with control and meta (OS)

Reply:
🙌